src/OpenColorIO/fileformats/xmlutils/XMLReaderUtils.h

```diff
@@ -164,7 +164,7 @@ void ParseNumber(const char * str, size_t startPos, size_t endPos, T & value)
 
     FindSubString(startParse, endPos - startPos, adjustedStartPos, adjustedEndPos);
 
-    const auto result = NumberUtils::from_chars(startParse + adjustedStartPos, startParse + adjustedEndPos, val);
+    const auto result = OCIO_NAMESPACE::from_chars(startParse + adjustedStartPos, startParse + adjustedEndPos, val);
```

Comment:
Based on the following discussion `OCIO_NAMESPACE::from_chars` must be `NumberUtils::from_chars`.
Note: The `OCIO_NAMESPACE` is useless because the code is already in this namespace.